examples/lib/defaultHtmlStepUi.js

```diff
@@ -155,10 +153,10 @@ function DefaultHtmlStepUi(_sequencer, options) {
                          </div>';
         $step('div.details').append(div);
       }
-      $step('div.panel-footer').append(
+      $step('div.panel-footer').append( // Save button.
         '<div class="cal collapse in"><button type="submit" class="btn btn-sm btn-default btn-save" disabled = "true" >Apply</button> <small style="padding-top:2px;">Press apply to see changes</small></div>'
       );
-      $step('div.panel-footer').prepend(
+      $step('div.panel-footer').prepend( // Tools: download and insert step.
```

Comment:
Tools or tool? Maybe I am wrong here.
Write a sentence instead of important words so that it is descriptive.